packages/file_selector/file_selector/example/lib/open_image_page.dart

```diff
@@ -15,8 +15,11 @@ class OpenImagePage extends StatelessWidget {
       label: 'images',
       extensions: <String>['jpg', 'png'],
     );
-    final List<XFile> files =
-        await openFiles(acceptedTypeGroups: <XTypeGroup>[typeGroup]);
+    final List<XFile> files = await openFiles(
+      acceptedTypeGroups: <XTypeGroup>[typeGroup],
+      initialDirectory: '/',
```

Comment:
This is not a very good example of initial directory; it would be better to make the example depend on `path_provider` and use something that could actually be a useful initial directory like the documents directory.